### ISSUE TITLE
ApiPopup Error Message is not visible due to color contrast

### ISSUE THREAD

**Issue by HritikR** [jackschedel/gpt.koaladev.io#68]
The error message in the API popup is not visible due to a color contrast problem.

![Screenshot 2023-08-13 at 9 13 21 PM](https://github.com/jackschedel/KoalaClient/assets/35923605/0be26f63-164b-45c9-9868-5bd758974b9e)
![Screenshot 2023-08-13 at 9 13 38 PM](https://github.com/jackschedel/KoalaClient/assets/35923605/04f04a62-849c-47c8-9941-560afb09c717)
---
It can be fixed by making the text color to white.

![Screenshot 2023-08-13 at 9 19 44 PM](https://github.com/jackschedel/KoalaClient/assets/35923605/11b2f7b1-9c35-409e-a4d5-c6cbbbacefe3)
![Screenshot 2023-08-13 at 9 20 10 PM](https://github.com/jackschedel/KoalaClient/assets/35923605/4be84c3e-873c-4f8a-9887-a97161d7214e)

**Comment by jackschedel** [comment]
good catch. theres probably a couple edge cases where I missed the color refactoring. just fixed it, I'm going to put it in a release with a few more color refactors
